galleries.py

```
@@ -19,8 +23,43 @@
+    Input:
+        url     Original photojournal URL w/PIA embedded, example:
+                https://photojournal.jpl.nasa.gov/catalog/PIA24615
+    """
+
+    # Only allow http/https schemes
+    if not url.startswith(('http://', 'https://')):
+        return url
+
+    try:
+        opener = urllib.request.build_opener()
+        request = urllib.request.Request(url)
```

Ruff (0.15.9)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)